api/graphql/schema.graphql

```diff
@@ -2280,7 +2284,6 @@ type Mutation {
     @create
     @canModel(ability: "create", injectArgs: true)
   updateCommunityInterest(
-    id: UUID!
```

Comment:
If the `id` argument is being removed, `testApplicantCanUpdateOwn` will need to be updated accordingly as it [fails](https://github.com/GCTC-NTGC/gc-digital-talent/actions/runs/13118855699/job/36599783342?pr=12668#step:7:61) looking for the `id` argument.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/4f15bf25afdce32a185fa3a8ed32733eb558bd95/api/tests/Feature/CommunityInterestTest.php#L110-L136

Reply:
Ah, right, forgot about that test.  Thanks.
[fix test](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12668/commits/c6e24f6cdf580b8e3c68464107383780c2710f51)